code/src/lib/Header.js

```diff
@@ -0,0 +1,38 @@
+import React from 'react'
+import { DateTimeNow } from './DateTimeNow.js'
+import { useSelector } from 'react-redux'
+import styled from 'styled-components/macro'
+
+
+export const Header = () => {
+  const todos = useSelector(state => state.todos.todos)
+  const list = useSelector(store => store.todos.list)
```

Comment:
Since you're only using the items-array, it's better to get only that here:
`const items = useSelector(store => store.todos.list.items)`